### ISSUE TITLE
Part segmentation

### ISSUE THREAD

**Issue by sebastianopazo1** [berkeley-hipie/HIPIE#10]
Hi, great job with the project. I have a question about part segmentation. I run successfully the demo, but the segmentation worked as sup-part segmentation. Is it possible to try it only with part segmentation?
Thanks!

**Comment by KKallidromitis** [comment]
Hi we currently support only part segmentation in the code base. The easiest approach is to replace the parts with a subpart dataset. Alternatively, you can use the custom demo and alter it for sub-parts and afterwards combine it with the normal demo.

**Comment by sebastianopazo1** [comment]
Hi! Thanks for the answer. What I meant was that as a result I'm getting a subpart result, just like the following image.
![output_image](https://github.com/berkeley-hipie/HIPIE/assets/53013616/99ccfb75-2a66-4d7a-a3c7-2d3b176874dd)
My question is if there is a way to make segmentation to the complete head instead of segmentating the head, nose and so on, for example.
Thanks for your help!

**Comment by KKallidromitis** [comment]
Hi there a couple ways to do this depending on the application. Existing model shown here is able to segment the head so you can remove the masks manually of the parts based on the label mapping and only keep the head (recommended). 

Another way to obtain the head is to use `torch.logical_or` on the different parts and combine the masks that give the head.
Finally we have a custom demo included where you are able to set your own labels. It will require some tweaking but you can obtain specific labels you request thought there.
